### PR TITLE
Fixes Issue#2301

### LINKED ISSUE #2301: Step 4 - Planet Tutorial (Make getting courses and upgrade clearer)
### Problem
For many people, the get courses button does not show, instead they can just go directly to the courses page. This should be clearly addressed

### Steps to reproduce the problem
N/A

### Screenshots
Screen Shot 2019-05-25 at 10 10 36 AM

### Proposed solution
Instead state:

1. Click the "Upgrade" button on Manager page and it take you to upgrade page. An update refers to a software update which improves the Planet (Note: If you do not see "Upgrade" button then simply carry on). It will then ask you for verification.
2. After the upgrade is complete, undergo `vagrant halt prod`, `vagrant up prod`, and log out and in again.
3. Go back to the Manager page and click on "Get Courses". (Note: If you do not see "Get Courses" button then simply carry on)

### PR DESCRIPTION
**Pull request by samuelchen1213**
Fixes #2301 

### Description
Made steps to "get courses" much clearer

### RawGit preview link
https://raw.githack.com/samuelchen1213/samuelchen1213.github.io/issue2301/#!pages/vi/vi-planetapps.md

### CODE REVIEW

**Comment by lmmrssa** [comment]
@samuelchen1213 You removed the step of **Manage Sync** also note. you need to update image to reflect latest version where we have **Fetch Item** instead of **Get Courses** and **Get Resources**

**Comment by samuelchen1213** [comment]
@lmmrssa On it right now!

File: pages/vi/vi-planetapps.md [hunk 1]
```diff
@@ -52,11 +52,13 @@ There are three important kinds of updates that you may receive on the community
 
 As you can see from the image below, there is an update ready to be downloaded. Usually, next to the update, you should also see publication(s) (we send to you from the nation) ready to be downloaded.
 
-![Update from the nation](images/vi-update-publication.png "Dashboard in your localhost")
+![Update from the nation](images/vi-planet-upgrade.png "Dashboard in your localhost")
```
Comment (lmmrssa) [inline pages/vi/vi-planetapps.md]: change label to **Manager Dashboard** instead of just **Dashboard** in both pictures

**Comment by samuelchen1213** [comment]
@lmmrssa Changed label!

**Pull request merged**